src/eng_front.c

```diff
@@ -75,6 +75,10 @@ static const ENGINE_CMD_DEFN engine_cmd_defns[] = {
 		"FORCE_LOGIN",
 		"Force login to the PKCS#11 module",
 		ENGINE_CMD_FLAG_NO_INPUT},
+	{CMD_CLAIM_MODULE_TOKEN_FIPS,
+		"CLAIM_MODULE_TOKEN_FIPS",
+		"Take responsibility that Module and Token are FIPS complient",
+		ENGINE_CMD_FLAG_NO_INPUT},
```

Comment:
Do we need an engine control command to turn this feature off (either with `ENGINE_CMD_FLAG_NUMERIC` or with a separate control command)?